chart/sns-connector/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+description: Invoke functions from an AWS SNS messages.
+name: sns-connector
+version: 0.0.1
+sources:
+- https://github.com/openfaas/faas-netes
+home: https://www.openfaas.com
+icon: https://raw.githubusercontent.com/openfaas/media/master/OpenFaaS_logo_stacked_opaque.png
+keywords:
+- openfaaspro
```

Comment:
Let's have fans in there too?

Reply:
Do you mean: aws?

Reply:
No, faas. But my autocorrect changed it.